engine/proofview.ml

```diff
@@ -39,7 +39,7 @@ let proofview p =
 
 let compact el ({ solution } as pv) =
   let nf c = Evarutil.nf_evar solution c in
-  let nf0 c = EConstr.(to_constr solution (of_constr c)) in
```

Comment:
Why is this not using the standard `nf` functions?

Reply:
I've no idea, when I wrote this code EConstr did not even exist, not to talk about the anomaly about ground terms...

Reply:
It comes from c8c8ccdaaffefdbd3d78c844552a08bcb7b4f915

Reply:
nf_evars on constr is the awkwardly-named nf_evars_universes